mergify_engine/github_events.py

```diff
@@ -263,12 +263,11 @@ async def push_to_worker(
         if event["repository"]["archived"]:
             ignore_reason = "repository archived"
 
-        elif event["action"] != "rerequested":
-            ignore_reason = f"check_suite/{event['action']}"
+        elif event[event_type]["app"]["id"] != config.INTEGRATION_ID:
+            ignore_reason = "not a mergify check_suite"
```

Comment:
I'm lacking context as to why we only process check suite from Mergify here?

Reply:
We use the check-suite/rerequested event to run the engine when people push the GitHub UI re-run button.

Reply:
I'd thought we wanted to also be triggered on `check_suite` `completed` event just in case?

Reply:
Hum, ok, that looks legit seatbelt.